libs/database/src/lib/entities/i18n-item.entity.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  Model,
+  PrimaryKey,
+  Table,
+} from "sequelize-typescript";
+import { BIGINT, STRING } from "sequelize";
+
+@Table({ tableName: "i18n_items", underscored: true, paranoid: false })
```

Comment:
`paranoid` tells sequelize to add/manage a `deleted_at` field, which this table doesn't have. Please remove `paranoid` from the table def here.